### DISCUSSION TITLE
如何解决某些页面（如某些 CSDN 页面或微信订阅号）等因结构太多特别导致解析效果不好的问题

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2561] [站点适配]
## 问题现象

例如某些奇葩的微信订阅号，如 https://mp.weixin.qq.com/s/gK4MhsfTSKNZmpeWnWSYzQ 进入阅读模式后会出现下面的情况，**也就是使用了阅读模式后反而效果不好**

![image](https://user-images.githubusercontent.com/81074/126489329-8ae75162-7d95-40a7-afcc-170326a79534.png)

## 解决方案

> 选项页 → 高级设定 → 词法分析引擎 → **关闭纯粹模式**，即可，如下图所示

![image](https://user-images.githubusercontent.com/81074/126436968-568b9132-6b09-4d96-a74b-a5c7eea9409e.png)

> 关闭后效果

![image](https://user-images.githubusercontent.com/81074/126592608-ee33bcb3-fe5c-44f2-a8c9-6b0082d9f93c.png)

## 发生问题的页面

几乎全部的页面均可开启纯粹模式，极个别页面存在此问题，大多数都发生在订阅号 e.g. https://github.com/Kenshin/simpread/issues/2554 https://github.com/Kenshin/simpread/issues/2556 等。

**Comment by JokAlec** [comment]
请帮我看看这个公众号文章，解析不出来（我已经关闭纯粹模式了）
https://mp.weixin.qq.com/s/2RJUnvZabOWk6SoOyg9U_Q
https://mp.weixin.qq.com/s/DwkkpAmr4InI1GW_Bs9WhQ

**Reply by Kenshin** [reply]
嗨，这应该是另外一个 Bug 造成的，只能下个版本解决。
